static/bidder-params/algorix.json

```diff
@@ -21,6 +21,10 @@
     "appId": {
       "type": "string",
       "description": "An ID which identifies this app of the impression"
+    },
+    "region": {
+      "type": "string",
```

Comment:
Would it be better to use an enum type here, like:
```
"enum": ["APAC", "USE"],
```
So that you can fail fast in case of an invalid region being passed in